__main__.py

```diff
@@ -0,0 +1,93 @@
+# Welcome to the WEAVER. This module......
+#
+#
+# Copyright (C) 2015 Weecology
+#
+# This module is part of weaver and is released under
+# the MIT License: http://www.opensource.org/licenses/mit-license.php
```

Comment:
We have tended to keep all of the copyright, license, and author level information at the level of the package rather than the individual file in the past. If there's a compelling reason to do this at the file level I'm happy to chat more about it. If not let's just keep things simpler by doing it at the repo level.